components/tabs/stories/tabs-horizontal.stories.js

```diff
@@ -30,11 +28,110 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
+const chromaticKitchenSink = ({
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; gap: 2rem;">
+    <h2>Horizontal Default</h2>
```

Comment:
If we're going to label items in the snapshots, can we run them through typography? That protects us from unexpected environment changes (font-size, typography, etc) from influencing all our snapshots. See button for some examples.